internal/command/action.go

```diff
@@ -200,3 +216,23 @@ func returnStatusOfError(err error) (int, error) {
 	}
 	return 0, err
 }
+
+// printProviderVersions returns a string of all provider versions
+func printProviderVersions()(string, error){
+	var providerVersions bytes.Buffer
+	
+	providers, err := prov.GetAllProviders()
+	if err != nil {
+		return "", err
+	}
+
+	for _, provider := range providers {
+		version, err := exec.Command(path.Join(prov.DefaultPath, provider), "--version").Output()
+		if err != nil {
+			providerVersions.WriteString(fmt.Sprintf("%s version uknown\n", provider))
+			continue
+		}
+		providerVersions.WriteString(fmt.Sprintf("%s version %s", provider, version))
```

Comment:
You might want to avoid depending on the provider to give you the `\n` character. I would probably opt that we strip the returned string and manually add the `\n` here just as you do for the unknown version string.